internal/restinterface/externalhandler/externalhandler.go

```diff
@@ -99,7 +100,7 @@ func init() {
 		restinterface.Route{
 			Name:        "APIV1RequestCloudSyncmgrGetSubscribedData",
 			Method:      strings.ToUpper("Get"),
-			Pattern:     "/api/v1/orchestration/cloudsyncmgr/getsubscribedata/{" + topic + "}/{" + clientID + "}",
+			Pattern:     "/api/v1/orchestration/cloudsyncmgr/getsubscribedata/{" + host + "}/{" + topic + "}/{" + clientID + "}",
```

Comment:
Is `clientID` correct instead  of `appID`?

Reply:
this is yet to be modified and shall be done in next PR. It shld be appid only

Reply:
The getSubscribeddata() API needs to be modified as per new hashmap{topic,url},[]clientid . So there will be few logic changes also like increase/decrease in number of arguments to be passed. Hence This change was postponed to nextPR to club all the chnages related to a API in one PR. No other specific reason for it